modules/streams/arrow-streams/src/test/kotlin/arrow/streams/internal/FreeCTest.kt

```diff
@@ -236,7 +236,7 @@ class FreeCTest : UnitSpec() {
     "Running a FlatMapped value using Either" {
       forAll(Gen.string()) { s ->
         FreeC.FlatMapped(FreeC.just("")) { FreeC.just<EitherPartialOf<Throwable>, String>(s) }
-          .run(Either.monadError()) == Right(Some(s))
+          .runFC(Either.monadError()) == Right(Some(s))
```

Comment:
we could also use invoke in these cases for Free. I think this is inline that both Free and Kleisli represent functions that you can apply to evaluate the structure.